harness/determined/deploy/gcp/terraform/modules/compute/main.tf

```diff
@@ -138,6 +140,7 @@ resource "google_compute_instance" "master_instance" {
         --name determined-master-configurator \
         --rm \
         -v /usr/local/determined/etc/:/etc/determined/ \
+        -v /home/danielrhunter/determined-master:/usr/bin/determined-master \
```

Comment:
Seems like this was accidentally committed.

Reply:
Oops!! Thanks for catching this.